### DISCUSSION TITLE
docs/upgrade-v2

### DISCUSSION THREAD

**Discussion by giscus** [zenstackhq/zenstack-docs#302] [Comments]
# docs/upgrade-v2

How to upgrade to ZenStack v2

https://zenstack.dev/docs/upgrade-v2

**Comment by tmax22** [comment]
if you have any issue after upgrading(as we had), make sure to delete any node_modules folder in your workspace, reinstalling dependencies, and rerun `npx zenstack generate`, this solved us `Cannot find module .zenstack/zod/models or its corresponding type declarations.` error.
